packages/multi-test/src/wasm.rs

```diff
@@ -861,8 +868,12 @@ mod test {
         WasmKeeper::new()
     }
 
-    fn mock_router() -> Router<Empty> {
-        Router::new(BankKeeper {})
+    fn mock_router() -> Router<Empty, Empty> {
+        Router {
+            wasm: Box::new(WasmKeeper::new()),
+            bank: Box::new(BankKeeper::new()),
+            custom: Box::new(MockSimpleCustomHandler::new()),
+        }
```

Comment:
Does this calls for using the builder pattern for `Router` too?

Reply:
I don't think it makes sense, as Router is basically internal tool. Here it is the only place it is constructed besides app, and it is for very specific test, but in general router should never be constructed on its own so I don't see too much profit from moving those 5 lines to external builder (which I don't want to expose publicly).